lib/protobuf/nats/config.rb

```diff
@@ -75,6 +77,18 @@ def new_tls_context
         tls_context
       end
 
+      def make_subscription_key_replacements(subscription_key)
+        service_replacements.each do |replacement|
```

Comment:
service replacements is an array that contains hashes.

[{"foo" => "bar"}, {"baz" => "qux"}] so you can't iterate over it that way.

```ruby
irb(main):001:0> test = [{"hello" => "world"}, {"foo" => "bar"}]
irb(main):002:0> test.each { |key, value| puts "key: #{key}"; puts "value: #{value}" }
key: {"hello"=>"world"}
value:
key: {"foo"=>"bar"}
value:
=> [{"hello"=>"world"}, {"foo"=>"bar"}]
```